<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 47, 48);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>290</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>290</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 8pt &quot;MS Shell Dlg 2&quot;;

</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="_2">
           <item>
            <widget class="QTextEdit" name="textEdit">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>100</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);


</string>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="download_pbh">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(89, 89, 70);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Настройки соединения</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Connect_DB">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 238, 0);
color: rgb(255, 255, 255);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Connect to DB</string>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
selection-background-color: rgb(158, 158, 158);
background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <property name="documentMode">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="publish_house">
           <attribute name="title">
            <string>Издательство</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="myQTableView" name="pbh_tableView">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Вид публикации</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="myQTableView" name="type_of_pb_tableView">
                <property name="styleSheet">
                 <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Главная</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="myQTableView" name="main_tableView">
                <property name="styleSheet">
                 <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
border-left-color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="append_row">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>28</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(89, 89, 70);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Добавить строку</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="insert_row">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>28</height>
             </size>
            </property>
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(89, 89, 70);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Вставить строку</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove_row">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>28</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(89, 89, 70);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Удалить строки</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1293</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myQTableView</class>
   <extends>QTableView</extends>
   <header>myqtableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
